moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1019,11 +1019,9 @@ const Eigen::Isometry3d& RobotState::getFrameInfo(const std::string& frame_id, c
     frame_found = true;
     return IDENTITY_TRANSFORM;
   }
-  if (robot_model_->hasLinkModel(frame_id))
+  if ((robot_link = robot_model_->getLinkModel(frame_id, &frame_found)))
   {
-    robot_link = robot_model_->getLinkModel(frame_id);
     BOOST_VERIFY(checkLinkTransforms());
-    frame_found = true;
```

Comment:
This boolean is used outside of the function and shouldn't be deleted. Apart from that this all looks good to me.

Reply:
Both, robot_link and frame_found are set in the above call to getLinkModel.

Reply:
I overlooked it, my bad. Good call.